### PR TITLE
Add Log In page & User Authentication#3

### PR DESCRIPTION
**Pull request by Dachrono**
# 🎉 🎈 In this *Milestone* 🙋‍♂️ 🚩 ‼ :

**Hi there** 👋

- Follow the best GitHub flow. ✅
- Follow our list of best practices for Ruby. ✅
- Document in a professional way. ✅
- No linters errors. ✅
- Create the Log In component and form. ✅
- Set the component as the main root in the App component. ✅
- Add the styles. ✅ 
- Add user authentication. ✅

# To the code reviewer 👨‍💻
I really thank 🙇‍♂️ you, dear code reviewer 👨‍🎨 for dedicating your time ⌚ to check the PR 🧾 .

I kindly ask code reviewer 🤵 that if there is any issue in this project, please do list 📃 them in a descriptive 💡 manner and give your best suggestions if needed.

If you think some significant issues need to be changed ♻️ please kindly contact 📞 me through Zoom or Slack, I am available on Microverse UTC-6 time zone.

### CODE REVIEW

File: src/components/LoginPage.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  };
+
+  const tryLogin = async (event) => {
+    event.preventDefault();
+    const result = dispatch(consultUser(inputValue));
+    if (result) {
+      dispatch(setUser(inputValue));
+      navigate('/MainPage');
+    }
+  };
```
Comment (AndreaM2429) [inline src/components/LoginPage.jsx:23]: Please kindly check this part of the code, when I set a non-existent username the page allows me access to the main page. The response of `result` is a promise so I suggest you return a correct response.

File: src/Redux/slices/userSlice.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    return Promise.reject(error);
+  }
+});
+
+const initialState = {
+  user: null,
+  userName: null,
+};
+
+const userSlice = createSlice({
```
Comment (AndreaM2429) [inline src/Redux/slices/userSlice.js:18]: Kindly remove this file. I can notice your effort and hard work on this implementation as it is needed to create and test the login page, however this `UserSlice` corresponds to another task, please don't get ahead with the tasks.

**Review by AndreaM2429 — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️ 

### Hi @Dachrono  

I can notice your effort in this task, however, there is something that needs to be fixed before continuing. Please check the comments below to know the required changes.

almost

## Highlights 

- Linters running
- Good styles
- Clean code

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval. Still, I would highly recommend that you consider these changes._

Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

----------------------------------------------------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/LoginPage.jsx (hunk 1 above)
Comment (Dachrono) [reply src/components/LoginPage.jsx:23]: Tky for your time and your comments, Solved

File: src/Redux/slices/userSlice.js (hunk 2 above)
Comment (Dachrono) [reply src/Redux/slices/userSlice.js:18]: Tky for your time and your comments, Solved

**Pull request closed without merging**